src/kiwixapp.cpp

```diff
@@ -113,6 +114,28 @@ void KiwixApp::init()
             m_library.asyncUpdateFromDir(dir);
         }
     }
+
+    /* Place session file in our global library path */
+    QDir dir(m_libraryDirectory);
+    this->mp_session = new QSettings(dir.filePath("user.session"),
+                                     QSettings::defaultFormat(), this);
+    QStringList tabsToOpen =
+        this->mp_session->value("reopenTabList", QStringList{})
```

Comment:
What happens if you don't provide `QStringList{}` as the default value? Won't a null `QVariant` object be converted to an empty `QStringList`?

Reply:
True, it is converted to emtpy QStringList. I will revert to default call.

Reply:
Done